### PR TITLE
introduce pull --include-deps

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
introduce pull --include-deps
disable all dependency services so that they are ignored by Pull

**Related issue**
https://github.com/docker/compose-cli/issues/1361

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: cli/cmd/compose/pull.go [hunk 1]
```diff
@@ -57,6 +59,15 @@ func runPull(ctx context.Context, opts pullOptions, services []string) error {
 		return err
 	}
 
+	if !opts.includeDeps {
+		enabled, err := project.GetServices(services...)
+		if err != nil {
+			return err
+		}
+		project.DisabledServices = append(project.DisabledServices, project.Services...)
```
Comment (gtardif) [inline cli/cmd/compose/pull.go:67]: DisabledServices will contain all services including the enabled ones here. Not affecting the current pull execution path probably, but very confusing, we should remove enabled services from there

**Review by gtardif — CHANGES_REQUESTED** [review]
LGTM except filling DisabledServices with some that are actually enabled

File: cli/cmd/compose/pull.go (hunk 1 above)
Comment (ndeloof) [reply cli/cmd/compose/pull.go:67]: ok, I'll fix the other places we do the same for "noDeps"

Comment (ndeloof) [reply cli/cmd/compose/pull.go]: see https://github.com/docker/compose-cli/pull/1377/commits/42d1dd2e0d0d7ce3d77e384445b79eb93905fda7

**Review by gtardif — APPROVED** [review]

**Pull request merged**